i18n/ui.json

```diff
@@ -83,6 +83,8 @@
   "resources": {
     "title": "Community Resources",
     "description": "A rich compilation of technical descriptions and detailed information of how Chakra UI works.",
+    "showcaseBannerTitle": "Visit the Showcase",
+    "showcaseBannerDescription": "See what other Devs made with Chakra UI. There are also some projects that provide you ready to use examples and some nice tools to e.g. help you define custom colors schemes for using in the theme.",
```

Comment:
"See what other Devs made with Chakra UI. Those are community projects, websites, libraries, tools, articles and videos that you can use to get inspired, implement a feature, or learn Chakra UI."

Reply:
Yes, thats better! Changed it.